webpack.config.js

```diff
@@ -101,7 +105,7 @@ module.exports = {
               importLoaders: 1,
               modules: {
                 auto: true,
-                localIdentName: '[path][name]__[local]--[hash:base64:5]',
+                localIdentName: '[path][name]__[local]--[contenthash:base64:5]',
```

Comment:
recommended change from [updating to webpack 5](https://webpack.js.org/migrate/5/)